restcomm/restcomm.http/src/main/java/org/restcomm/connect/http/CallsEndpoint.java

```diff
@@ -386,16 +405,17 @@ protected Response putCall(final String accountSid, final MultivaluedMap<String,
         }
 
         CreateCall create = null;
+
         try {
             if (to.contains("@")) {
```

Comment:
why when To header contains "@", custom headers are not supported?

Reply:
@jaimecasero this is a call to a SIP URI and custom header are supposed to be already in the SIP URI if needed